### PR TITLE
Bump sbt-org-policies to 0.7.4

### PR DESCRIPTION
**Pull request by BenFradet**
It also migrates from cats `0.9.0` to `1.0.0-MF`, among other upgrades and enhancements.

### CODE REVIEW

**Comment by BenFradet** [comment]
Thanks, I wasn't entirely sure sbt-org-policies needed to be involved, I'll take care of https://github.com/47deg/sbt-org-policies/issues/538 as well then.

**Comment by juanpedromoreno** [comment]
@BenFradet I needed to upgrade sbt-org-policies and btw I upgraded everything in general: https://github.com/47deg/sbt-org-policies/pull/632, including Scalajs.

**Comment by BenFradet** [comment]
Thanks a lot! I'll bump sbt-org later today :+1:

**Comment by BenFradet** [comment]
mmmh getting https://travis-ci.org/47deg/github4s/jobs/276970366#L846 with 0.6.4 and sbt doesn't launch with 0.7.0

**Comment by suhasgaddam** [comment]
@BenFradet I think Scala.js `0.6.20` requires sbt `0.13.16`.
with
`sbt-org-policies`:`0.7.0`
`build.properties`: `sbt.version = 0.13.16`

`sbt +compile` seems to work for me, locally

Links that I stumbled on and for anyone else searching:
https://stackoverflow.com/questions/46206462/java-lang-nosuchmethoderror-sbt-package-zerolsbt-global
http://www.scala-js.org/news/2017/09/01/announcing-scalajs-0.6.20/ -> `Breaking Changes`

**Comment by suhasgaddam** [comment]
The travis ci error: `sbt-org-policies`:`0.6.4` did not have `sbt-microsites` enabled. We are/were still working on getting `sbt-microsites` cross-compiling with sbt 1.0.

**Comment by BenFradet** [comment]
@suhasgaddam thanks! didn't know about the issue with microsites / 0.13.16 min for 0.6.20

File: project/plugins.sbt [hunk 1]
```diff
@@ -1,2 +1,3 @@
+addSbtPlugin("com.eed3si9n" % "sbt-buildinfo" % "0.7.0")
 addSbtPlugin(
-  "com.47deg" % "sbt-org-policies" % "0.5.4" exclude ("io.get-coursier", "sbt-coursier"))
+  "com.47deg" % "sbt-org-policies" % "0.6.4" exclude ("io.get-coursier", "sbt-coursier"))
```
Comment (juanpedromoreno) [inline project/plugins.sbt]: It should be `0.7.0`

Comment (BenFradet) [reply project/plugins.sbt]: fixed :fearful:

**Comment by BenFradet** [comment]
Weirdly enough it seems to compile fine even though `cats.data.Coproduct` was removed from cats-1.0.0-MF.

However, it fails during the cats-effect tests with https://travis-ci.org/47deg/github4s/jobs/277197841#L2596.

Still investigating :confused:

**Comment by BenFradet** [comment]
@juanpedromoreno fixed! :+1: Regarding releasing a version, do I just need to edit `version.sbt`? Should I introduce a specific commit?

**Comment by juanpedromoreno** [comment]
Yes, you only need to modify the `version.sbt`.

Are we releasing the `0.1.5.1` or `0.16.0`? As it's not binary compatible I propose `0.16.0`, what do you think?

Either way, I'd suggest doing it in a different PR, since due to https://github.com/47deg/sbt-org-policies/issues/637, the plugin is not running the release task correctly. Hence, we might need to edit the `orgAfterSuccessCI` task by something like this https://github.com/47deg/sbt-org-policies/blob/2c89b98d9366b626a978e3ff17cbc25e561ea48b/.travis.yml#L31-L44, in the `.travis.yml` file.

**Comment by BenFradet** [comment]
@juanpedromoreno ok thanks :+1: , I'll open a new PR

**Pull request merged**